src/solvers/solvers_options.jl

```diff
@@ -22,7 +22,7 @@ Creates an object `PG` containing the options of the Proximal Gradient solvers:
 
 """
 struct PG <: ForwardBackwardSolver
-    kwargs::Array
+    kwargs::Iterators.Pairs
```

Comment:
I guess this is how kwargs are splatted now... but it used to be an Array of pairs?? Crazy

Reply:
apparently yes! :smile: